db/migrations/state/0023.sql

```diff
@@ -1,18 +1,57 @@
 -- +migrate Up
 
-
--- Add first blob_sequence?
-CREATE TABLE state.blob_sequence
+CREATE TABLE IF NOT EXISTS state.blob_sequence
 (
     index                BIGINT PRIMARY KEY,
     coinbase             VARCHAR,
     final_acc_input_hash VARCHAR,
     first_blob_sequenced  BIGINT,
     last_blob_sequenced   BIGINT,
     created_at           TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    recevied_at          TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
     block_num          BIGINT NOT NULL REFERENCES state.block (block_num) ON DELETE CASCADE 
 );
 
+comment on column state.blob_sequence.index is 'Is a id of this sequence, this value is internal and incremental';
+comment on column state.blob_sequence.block_num is 'L1 Block where appear this sequence';
+comment on column state.blob_sequence.first_blob_sequenced is 'first (included) blob_inner_num of this sequence (state.blob_inner.blob_inner_num)';
+comment on column state.blob_sequence.first_blob_sequenced is 'last (included) blob_inner_num of this sequence (state.blob_inner.blob_inner_num)';
+comment on column state.blob_sequence.recevied_at is 'time when it was received in node';
+comment on column state.blob_sequence.created_at is 'time when was created on L1 (L1block tstamp)';
+
+CREATE TABLE IF NOT EXISTS state.blob_inner_in 
```

Comment:
What about just name it as state.blob_inner?

Reply:
This name was used by Aggregator branch, until we don't merge both changes and understand if it's the same I think that is better to keep separated